<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>298</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Metronome</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="OutputGBox">
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>BPM:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="bpmOut">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Bars:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="barOut">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Beats:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="beatOut">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="BeatGBox">
       <property name="title">
        <string>Beat</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="minimum">
             <number>4</number>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Beats per bar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="fourRB">
            <property name="text">
             <string>/4</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="sixteenRB">
            <property name="text">
             <string>/16</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="eightRB">
            <property name="text">
             <string>/8</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="thirtyTwoRB">
            <property name="text">
             <string>/32</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="mainLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>BPM</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="bpLayout">
           <item>
            <widget class="QSpinBox" name="bpmSB">
             <property name="font">
              <font>
               <pointsize>36</pointsize>
              </font>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>40</number>
             </property>
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="value">
              <number>120</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="Volume">
           <property name="value">
            <number>75</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="speed_trGB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Speed Training</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="en_sp_trRB">
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="add_bpmSB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>BPM add</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSpinBox" name="barsLimitSB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Bars per step</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start/Stop</string>
    </property>
    <property name="icon">
     <iconset theme="play">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>188</y>
      <width>471</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="checkBox_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_8">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QCheckBox" name="checkBox_9">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QCheckBox" name="checkBox_13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QCheckBox" name="checkBox_15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_16">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QCheckBox" name="checkBox_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QCheckBox" name="checkBox_17">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_18">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QCheckBox" name="checkBox_21">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_22">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <item>
        <widget class="QCheckBox" name="checkBox_31">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_32">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QCheckBox" name="checkBox_19">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_20">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QCheckBox" name="checkBox_25">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_26">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QCheckBox" name="checkBox_29">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_30">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QCheckBox" name="checkBox_23">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_24">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QCheckBox" name="checkBox_27">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_28">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_25">
       <item>
        <widget class="QCheckBox" name="checkBox_41">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_42">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_28">
       <item>
        <widget class="QCheckBox" name="checkBox_47">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_48">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_34">
       <item>
        <widget class="QCheckBox" name="checkBox_59">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_60">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_26">
       <item>
        <widget class="QCheckBox" name="checkBox_43">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_44">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_31">
       <item>
        <widget class="QCheckBox" name="checkBox_53">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_54">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_32">
       <item>
        <widget class="QCheckBox" name="checkBox_55">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_56">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_30">
       <item>
        <widget class="QCheckBox" name="checkBox_51">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_52">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_33">
       <item>
        <widget class="QCheckBox" name="checkBox_57">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_58">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <widget class="QCheckBox" name="checkBox_33">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_34">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_23">
       <item>
        <widget class="QCheckBox" name="checkBox_37">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_38">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_27">
       <item>
        <widget class="QCheckBox" name="checkBox_45">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_46">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_24">
       <item>
        <widget class="QCheckBox" name="checkBox_39">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_40">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_36">
       <item>
        <widget class="QCheckBox" name="checkBox_63">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_64">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_29">
       <item>
        <widget class="QCheckBox" name="checkBox_49">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_50">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_35">
       <item>
        <widget class="QCheckBox" name="checkBox_61">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_62">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_22">
       <item>
        <widget class="QCheckBox" name="checkBox_35">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_36">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>en_sp_trRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>add_bpmSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>en_sp_trRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>en_sp_trRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>barsLimitSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>en_sp_trRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
